utils/assetChecks.ts

```diff
@@ -70,3 +70,112 @@ export const isAsset = (asset?: Asset | null) =>
  */
 export const isCollection = (asset?: Asset | null) =>
   isLsp8(asset) && !!asset?.tokenIdsData?.length
+
+/**
+ * Check if passed asset has balance
+ *
+ * @param asset
+ * @returns
+ */
+export const hasBalance = (asset?: Asset | Profile | null) =>
+  !!asset?.balance && asset?.balance !== '0'
```

Comment:
Should not be possible. The values in graphql are numeric inside of postgres.

Reply:
Nice to know! Thanks!

Reply:
Also keep in mind that we not just deal with indexed data, for instance LYX balance we fetch directly from chain. Another factor is the web3 version, current one return balance as a string but in latest versions use Big Number. Thats why we use this small wrapper functions to get or check balance because if format changes we just need to update one place.